plugins/amdgpu/dummy_plugin.c

```diff
@@ -0,0 +1,51 @@
+#include <errno.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+
+#include <linux/limits.h>
+
+#include <sys/ioctl.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <stdint.h>
+
+#include "criu-log.h"
+#include "criu-plugin.h"
+
+#include "common/list.h"
+
```

Comment:
Are all those includes really needed?

Reply:
I think criu-plugin.h is needed but I will recheck. Will remove list.h

Reply:
I would say most of them are unneeded.